parsing/lexer.mll

```diff
@@ -413,7 +413,11 @@ rule token = parse
         let s, loc = with_comment_buffer comment lexbuf in
         COMMENT (s, loc) }
   | "(*" ('*'*) as stars "*)"
-      { COMMENT (stars, Location.curr lexbuf) }
+      { if !handle_docstrings && stars="(*" then
```

Comment:
After a quick look, except for two cases in `lex/lexer.mll` line 229 and `manual/tools/transf.mll` line 45, all other uses of `as` in lexers are of the form `( ... as ...)` which should be safe?